static/js/gio_calendar_next_cm.js

```diff
@@ -19,23 +19,33 @@ function retrieveNextCommunityMeeting() {
       // check if various properties exist, and set variables if so
       var timeRange = e.pretty_time ? `<time><i class="fa fa-time"></i> ${e.pretty_time}</time><br>` : '';
```

Comment:
is this time fa icon showing either? hmm

Reply:
This fa icon needed to be updated to show (see screenshot above) but I don't know that it's necessary. It's not being used on the GIO page. I removed it, but we can add it back if we think it's valuable.

Reply:
![Screen Shot 2021-04-29 at 8 59 59 AM](https://user-images.githubusercontent.com/13525011/116563290-83ea5f00-a8c9-11eb-8473-0b152f34e7a7.png)

it's in the code for the gio page, but not showing

Reply:
i would vote just to remove it

Reply:
It could be updated to `<i class="far fa-clock"></i>` but I was thinking of removing it there too, unless we think it's necessary.